packages/shared/src/components/sidebar/sections/CustomFeedSection.tsx

```diff
@@ -58,32 +57,29 @@ export const CustomFeedSection = ({
 
     return [
       ...customFeeds,
-      showPlusSubscription
-        ? {
-            icon: () => (
-              <div className="rounded-6 bg-background-subtle">
-                <PlusIcon />
-              </div>
-            ),
-            title: 'Custom feed',
-            path: `${webappUrl}feeds/new`,
-            requiresLogin: true,
-            isForcedClickable: true,
-          }
-        : undefined,
+      {
+        icon: () => (
+          <div className="rounded-6 bg-background-subtle">
+            <PlusIcon />
+          </div>
+        ),
+        title: 'Custom feed',
+        path: `${webappUrl}feeds/new`,
+        requiresLogin: true,
+        isForcedClickable: true,
+      },
     ].filter(Boolean);
   }, [
     defaultRenderSectionProps.activePage,
     feeds?.edges,
     defaultFeedId,
     onNavTabClick,
-    showPlusSubscription,
   ]);
 
   /**
    * If there are no custom feeds and the user is not subscribed to Plus don't show this section
    */
-  if (!menuItems.length && !showPlusSubscription) {
+  if (!menuItems.length) {
     return null;
   }
```

Comment:
I think that we should remove this if at all, condition were: `If there are no custom feeds AND the user is not entrolled to Plus`

Reply:
Makes sense! Yes, let's remove this one.